python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -342,6 +342,8 @@ def get_cam_type_and_inst_name(self, inst: str) -> tuple[CamType, str]:
             return CamType.LsstCam, "lsst"
         elif inst == "lsstfam":
             return CamType.LsstFamCam, "lsstfam"
+        elif inst == "comcam":
```

Comment:
Do you want to add `comcam` to a test of the `get_cam_type_and_inst_name` function ?

Reply:
Added. And added just went ahead and added fam since it wasn't in there either.